google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -127,6 +129,17 @@ final class StreamingSubscriberConnection extends AbstractApiService implements
   private OpenTelemetryPubsubTracer tracer = new OpenTelemetryPubsubTracer(null, false);
   private final SubscriberShutdownSettings subscriberShutdownSettings;
 
+  private final boolean enableKeepalive;
+  private static final long KEEP_ALIVE_SUPPORT_VERSION = 2;
```

Comment:
I believe that we are considering `protocol_version==1` as the minimum protocol version to support the keep alive ping.